Dangerfile

```diff
@@ -27,8 +27,12 @@ end
 
 message "Pull Request is marked as Work in Progress" if github.pr_title.include? "[WIP]"
 
+junit.parse "output2.xml"
```

Comment:
ah ok. But then you could skip this part and have the original tests put in `/spec/TEST-Teaspoon-Result.xml` right?

Reply:
Ok I will directly parse /spec/TEST-Teaspoon-Result.xml